SlackLogger/LoggerProvider.cs

```diff
@@ -13,16 +12,10 @@ internal class LoggerProvider : ILoggerProvider
         private readonly SlackLoggerOptions _options;
         private readonly ScopeSettings _scopeSettings;
 
-        public LoggerProvider(IOptions<LoggerFilterOptions> filterOptions, IOptions<SlackLoggerOptions> options, IConfiguration configuration, IHostingEnvironment env)
+        public LoggerProvider(IOptions<LoggerFilterOptions> filterOptions, IOptions<SlackLoggerOptions> options, IConfiguration configuration)
         {
             _options = options.Value;
             _options.Merge(configuration.GetSection("Logging:Slack"));
-            
-            _options.EnvironmentName = env?.EnvironmentName;
```

Comment:
Is it possible to make it work in ASP.Core without setting up EnvironmentName setting?
Maybe pulling the environment name from `Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT")` would work? Did not test it.

Reply:
Yes, I could add that